src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -48,6 +49,7 @@
 @dataclass
 class LoggerOperations(LogContext):
     stream: bool = False
+    run_mode: Optional[RunMode] = RunMode.Test
```

Comment:
Seems like we already have `run_nomde` in parent class.
![image](https://github.com/microsoft/promptflow/assets/17527303/18c1e9a1-6b0e-4b55-a2ea-bf5a52aeea9f)

Reply:
Updated.